src/manifest.json

```diff
@@ -166,6 +164,13 @@
       "matches": ["*://*/*"],
       "run_at": "document_start",
       "js": ["js/contentScriptOnStart.bundle.js"]
+    },
+    {
+      "allFrames": true,
```

Comment:
there is an inconsistency here with the entry above which has "all_frames": true,

Reply:
i just fixed that :+1: